checkov/github_actions/graph_builder/local_graph.py

```diff
@@ -105,11 +105,7 @@ def _create_permissions_vertices(self, file_path: str, permissions: Any) -> None
 
         if permissions is None:
             # if 'permissions' is not set in a file, then it is automatically 'write-all'
-            permissions = {
-                "permissions": "write-all",
-                START_LINE: 0,
-                END_LINE: 0,
-            }
+            permissions = "write-all"
```

Comment:
For GHA violations we rely on the start/end lines being zero to adjust the correct lines from the definitions.
Did you check that it is not affected by this?

Reply:
this was kind of redundant, because 4 lines below it will be added :smile: